cloudinit/config/cc_ntp.py

```diff
@@ -397,7 +421,14 @@ def write_ntp_config_template(distro_name, servers=None, pools=None,
     if not pools:
         pools = []
 
-    if len(servers) == 0 and len(pools) == 0:
+    if (len(servers) == 0 and distro_name == 'alpine' and
```

Comment:
Should we detect busybox or hard-code Alpine?

Reply:
When distro_name = 'alpine' and service_name = 'ntpd' that means Busybox's ntpd is in use. Other distros in cloud-init are not (currently) using Busybox's ntpd but rather openntpd so this logic is only specific to Alpine (at least for now).